severity.go

```diff
@@ -0,0 +1,51 @@
+package log
+
+import (
+	"fmt"
+	"strings"
+	"sync/atomic"
+)
+
+type Severity int32
+
+const (
+	SeverityInfo Severity = iota
+	SeverityWarning
+	SeverityError
+)
+
+var severityName = map[Severity]string{
+	SeverityInfo:    "INFO",
+	SeverityWarning: "WARN",
+	SeverityError:   "ERROR",
+}
+
+func (s *Severity) Get() Severity {
+	return Severity(atomic.LoadInt32((*int32)(s)))
+}
+
+func (s *Severity) Set(val Severity) {
+	atomic.StoreInt32((*int32)(s), int32(val))
+}
+
+func (s *Severity) Gte(val Severity) bool {
+	return s.Get() >= val
+}
```

Comment:
`Get`, `Set`, and `Gte` can all be deleted. The first two are no longer used anywhere, and the last one can be replaced with `s >= val`, since severities no longer change.